integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingTest.java

```diff
@@ -113,27 +113,6 @@ public void shouldNotConsumeMessagesWhenSubscriptionIsSuspended() {
     }
 
     @Test
-    public void shouldSendPendingMessagesAfterSubscriptionIsResumed() {
```

Comment:
This behaviour doesn't hold anymore. Main reason is because when you suspend a subscription entire consumer process is killed by a supervisor. There are no pending messages as a result. When you resume a consumer by default current offset is set to the largest value. If someone wants to receive events from the past those should be retransmitted.